app/models/ucsd_agent.rb

```diff
@@ -0,0 +1,30 @@
+# UcsdAgent is a subclass of EDM Agent (http://www.europeana.eu/schemas/edm/Agent)
+class UcsdAgent < Agent
+
+  configure type: ::UcsdTerms.UcsdAgent
+
+  property :agent_type, predicate: ::UcsdTerms.agent_type, multiple: false
+  property :has_orcid, predicate: ::RDF::URI.new("http://id.loc.gov/vocabulary/identifiers#orcid"), multiple: false
```

Comment:
@mcritchlow We could totally just use `orcid`. I was coming from a "be more explicit and include a verb" angle but realize that can sometimes instead create confusion.